staltz/ssb-fixtures
# support ssb-tribes

@arj03 @cryptix @mixmix I think fixtures now can generate: (1) tribe creation, (2) tribe invitation to other feeds, (3) post a message inside tribes, all using `ssb-tribes`. *I think* it's working, but it would be good to be sure.

How to test this out: 

- `git clone`
- checkout branch `tribes`
- `npm install`
- `npm run compile`
- `node lib/bin.js`
- check `./data` dir when it's done

Note, you can get the `secret` of the 3 most popular feeds, they are in the generated `./data/`.
